docker/ftest-jpa/src/test/resources/arquillian.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<arquillian xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="http://jboss.org/schema/arquillian"
+  xsi:schemaLocation="http://jboss.org/schema/arquillian
+    http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+
+  <extension qualifier="docker">
+    <property name="autoStartContainers">${arquillian.cube.autostart}</property>
+    <property name="definitionFormat">CUBE</property>
+    <property name="dockerContainers">
+      wildfly:
+        buildImage:
+          dockerfileLocation: src/test/resources/wildfly
+          noCache: true
+          remove: true
+        exposedPorts: [8080/tcp, 9990/tcp]
+        await:
+          strategy: polling
+        portBindings: [8081->8080/tcp, 9991->9990/tcp]
+        links:
+          - postgres:postgres
+      postgres:
+        image: postgres:9.3
+        exposedPorts: [5432/tcp]
+        await:
+          strategy: polling
```

Comment:
With PostgresSQL this strategy does not work always becauase PostgreSQL starts the server once, open the port, and then they run some process which closes the port and open again after running the process (it takes like 1 sec) so if you try to access to the postgres during this second then you will get a connection refused. 

http://www.lordofthejars.com/2017/05/testing-dockerized-sql-databases.html